content/desktop/release-notes.md

```diff
@@ -24,11 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+# 4.24.0
+
+{{< release-date date="2023-09-28" >}}
+
+{{< desktop-install all=true version="4.24.0" build_path="/" >}}
+
+### New
+
+- New Notification Center available to all users.
+- Resource Saver available to all users.
```

Comment:
```suggestion
- Resource Saver available to all users. This feature can be configured from the **Resources** tab in **Settings**. For more information see [Docker Desktop's Resource Saver mode](resource-saver.md)
```

Reply:
the resource-saver.md page is added in @ctalledo's pr [here](https://github.com/docker/docs/pull/18259/files#diff-6927c60a5fb3d56266d9d4f69068a8465f29335cf468ac4f2aff30a4212af419)

Reply:
done!